static/app/views/performance/transactionSummary/transactionAnomalies/content.tsx

```diff
@@ -174,24 +186,17 @@ type DataType = {
   chart: AnomalyData;
 };
 
-function Anomalies(props: Props) {
+function Anomalies(props: AnomaliesSectionProps) {
   const height = 250;
   const chartColor = theme.charts.colors[0];
 
   const chart = useMemo<QueryDefinition<DataType, WidgetDataResult>>(() => {
     return {
       fields: '',
-      component: provided => (
-        <AnomaliesQuery
-          orgSlug={props.organization.slug}
-          location={props.location}
-          eventView={props.eventView}
-          {...pick(provided, 'children')}
-        />
-      ),
+      component: provided => <Fragment>{provided.children(props.queryData)}</Fragment>,
       transform: transformAnomalyData,
     };
-  }, []);
+  }, [props.eventView, props.queryData]);
```

Comment:
This will be changed in a follow-up PR since I'll likely be writing a cross-page cache for this query since it's long.